MekHQ/data/universe/academies/Prestigious Academies.xml

```diff
@@ -0,0 +1,940 @@
+        <curriculum>Small Arms, Artillery, Anti-Mech, Leadership, Strategy, Tactics</curriculum>
+        <qualificationStartYear>2400</qualificationStartYear>
+        <qualification>Aerospace Officer Graduate</qualification>
+        <curriculum>Gunnery/Aerospace, Piloting/Aerospace, Leadership, Strategy, Tactics</curriculum>
+        <qualificationStartYear>2439</qualificationStartYear>
+        <baseAcademicSkillLevel>0</baseAcademicSkillLevel>
+    </academy>
+    <academy>
+        <name>Armstrong Flight Academy</name>
+        <isMilitary>true</isMilitary>
+        <promotion>Corporal</promotion>
+        <description>The premier flight school of the Federated Suns and a leading institution in the Inner Sphere, well known for producing top-tier aerospace pilots and naval technicians. Strategically co-located at Galax near Federated-Boeing Interstellar, the largest spacecraft manufacturer in the region, the academy benefits from close interactions with leading engineers and test pilots. This relationship enhances practical training, allowing the academy to concentrate on high-level piloting and technician skills.</description>
+        <faction>FC</faction>
+        <locationSystem>Galax</locationSystem>
+        <constructionYear>2790</constructionYear>
+        <destructionYear>3069</destructionYear>
+        <tuition>14000.</tuition>
+        <durationDays>900</durationDays>
+        <facultySkill>4</facultySkill>
+        <educationLevelMin>1</educationLevelMin>
+        <educationLevelMax>2</educationLevelMax>
+        <ageMin>16</ageMin>
+        <qualification>Aerospace Graduate</qualification>
+        <curriculum>Gunnery/Aerospace, Piloting/Aerospace</curriculum>
+        <qualificationStartYear>2439</qualificationStartYear>
+        <qualification>Naval Graduate</qualification>
+        <curriculum>Gunnery/Spacecraft, Piloting/Spacecraft, Tech/Vessel</curriculum>
+        <qualificationStartYear>2470</qualificationStartYear>
+        <qualification>Interstellar Technician Graduate</qualification>
+        <curriculum>Tech/Aero, Tech/Vessel</curriculum>
+        <qualificationStartYear>2470</qualificationStartYear>
+        <baseAcademicSkillLevel>0</baseAcademicSkillLevel>
+    </academy>
+    <academy>
+        <name>Armstrong Pheonix Academy</name>
+        <isMilitary>true</isMilitary>
+        <promotion>Corporal</promotion>
+        <description>Previously named the Armonstrong Flight Academy, the academy has been relocated and renamed Armstrong Phoenix Academy due to its original campus being rendered uninhabitable by a blood destroying bioweapon. Despite this, it maintains rigorous standards, with intense physical screenings and high skill requirements, leading to over half of the applicants being rejected pre-examination.</description>
+        <faction>FC</faction>
+        <locationSystem>Delavan</locationSystem>
+        <constructionYear>2070</constructionYear>
```

Comment:
Should this be 2070?

Reply:
Definitely not, there is also a typo in description. Will fix on Monday